mlflow/llama_index/tracer.py

```diff
@@ -95,25 +120,36 @@ def prepare_to_exit_span(
     ) -> _LlamaSpan:
         try:
             llama_span = self.open_spans.get(id_)
+            if not llama_span:
+                return
```

Comment:
Should we have some warnings here? Was it exited accidentally?

Reply:
Yea similar to LangChain we need to prepare for such unexpected situation. But I think we don't need to raise warning here, there is nothing users can do with it.